crates/torii/migrations/20230316154230_setup.sql

```diff
@@ -10,14 +10,13 @@ CREATE TABLE worlds (
     world_class_hash TEXT,
     executor_address TEXT,
     executor_class_hash TEXT,
-    created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP
```

Comment:
I'm not sure we want to remove these and `updated_at`. They can still be useful internally for debugging. We should not expose them publicly over the api

Reply:
Oh right, thought we would remove them. I'll re-add this!